review/views.py

```diff
@@ -1,3 +1,98 @@
-from django.shortcuts import render
+import json
+from django.views import View
+from django.http import request,JsonResponse
+from user.models import User
+from .models import Question
+from .models import QuestionType
+from .models import AnswerStatus
+from product.models import Product
+from datetime import datetime
+from user.utils import check_user
+
+class QuestionView(View):
```

Comment:
슬제님 QuestionView <-- class 명 깔끔하고 가독성 좋게 잘 만들어주셨습니다!!
질문 리스트를 반환하는 class 이므로 QuestionsView로 수정해주세요~
```suggestion
class QuestionsView(View):
```